app/models/user.rb

```diff
@@ -1,4 +1,9 @@
 class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :validatable, :confirmable
```

Comment:
- [ ] Nice work making the changes requested by the previous reviewer :clap: However, your App raise an error (see image) when registering a new user. Kindly make sure a user can register successfully without any error.
![Screenshot from 2023-12-21 14-35-08](https://github.com/ismayito/Rails-blog-app/assets/91142432/6e40a5da-8f98-4c9f-baa7-4a8b7031486e)